src/main/java/epicsquid/roots/entity/ritual/EntityRitualBase.java

```diff
@@ -25,6 +28,13 @@ public void setPosition(double x, double y, double z) {
     this.z = z;
   }
 
+  @Nonnull
+  @Override
+  public BlockPos getPosition()
```

Comment:
I added this because I needed to get the center position of the ritual, but I don't know if there's a better way to do it 🤔